website/src/libs/api-v3/setupRoutes.js

```diff
@@ -4,7 +4,11 @@ import express from 'express';
 import _ from 'lodash';
 
 const CONTROLLERS_PATH = path.join(__dirname, '/../../controllers/api-v3/');
-let router = express.Router(); // eslint-disable-line new-cap
+let router = express.Router(); // eslint-disable-line babel/new-cap
+
+// Wrapper function to handler `async` route handlers that return promises
+// It takes the async function, execute it and pass any error to next (args[2])
+let _wrapAsyncFn = fn => (...args) => fn(...args).catch(args[2]);
```

Comment:
Do we need to wait for this to be merged before we can start converting the controllers to the async syntax?